### PR TITLE
655 edits CTA, changes copy and updates pricing

### PR DESCRIPTION
**Pull request by isoworg**

### CODE REVIEW

**Comment by pitchinvasion** [comment]
@isoworg  the code is fine, did you still want nikesh/ruben to improve the copy?

**Comment by isoworg** [comment]
I'd like Nikesh and Ruben to check I've made all the changes they requested.

**Comment by nikeshashar** [comment]
image

This is the bit I flagged & it's been changed. 👍

**Comment by nikeshashar** [comment]
"We find exceptional people for you, and then train them." > "We find exceptional people for you **who want to learn to code**, and then train them."

(SEO improvement)

File: source/employers.html.haml [hunk 1]
```diff
@@ -63,7 +61,7 @@
         %p Once we've done our matching, you can follow your usual process, whether that's a tech test, a pairing interview or drinks at your place.
       .column
```
Comment (nikeshashar) [inline source/employers.html.haml]: Perhaps add a link to the blog post about what FNZ did. Replacing "drinks at your place" to "host a hackathon"?

**Comment by nikeshashar** [comment]
Added my comments inline too. 

Generally fine. I think the last panel should lead to a CTA like start hiring (maybe move the button from the panel above to the last one?

**Comment by isoworg** [comment]
Thanks @nikeshashar - not sure we have enough traffic to be moving elements 2 inches down the page, but we'll take it into account for the next iteration.

**Comment by isoworg** [comment]
Opening new issue with page improvements.

**Pull request merged**